internal/account/idgenerator.go

```diff
@@ -0,0 +1,24 @@
+package account
+
+import (
+	"crypto/rand"
+	"math/big"
+	"strings"
+)
+
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+func generateRandomID() (string, error) {
```

Comment:
Why do we have to generate random IDs manually? Why not use something like uuid?

Reply:
There are many ID generator packages out there some of which adhere to RFC standards. For example, we can use something like [KSUID](https://github.com/segmentio/ksuid), which sorts by generation timestamp

Reply:
I wanted to use uuid at first, however since the ID field in the Account table is varchar(15), we can't use UUID here. The reason why is the ID is varchar(15) is to make it consistant with other previous made table like Vendor and Product. Is KSUID support to generate ID that fits 15 characters? If yes, we can use that one.

Reply:
Alas! I did not realize that UUID or KSUID is not backward compatible with our (their) current database. How unfortunate! Therefore, I approve this MR. Good job